src/mon/OSDMonitor.cc

```diff
@@ -1076,7 +1076,10 @@ void OSDMonitor::prime_pg_temp(
   const OSDMap& next,
   pg_t pgid)
 {
-  if (mon->monmap->get_required_features().contains_all(
+  if (mon->monmap->get_required_features().empty()) {
+    dout(10) << __func__ << ": quorum not formed" << dendl;
+    return;
+  } else if (mon->monmap->get_required_features().contains_all(
```

Comment:
How are we even getting into these functions without a quorum? I don't think we should have to run those checks here (although we may need to handle stale results; I'm looking at that now).